src/main/resources/jenkins/plugins/openstack/compute/auth/AbstractOpenstackCredential/config.jelly

```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
+	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+</j:jelly>
```

Comment:
Do we need a `config.jelly` for the abstract class if it is overridden by all implementations?